### PR TITLE
Create publish.yml

### PR DESCRIPTION
**Pull request by HerrNaN**
Should publish to AUR on push to release tags

### CODE REVIEW

File: .github/workflows/publish-aur.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download archive
+        run: wget https://github.com/sidusio/boogrocha/archive/${GITHUB_REF##*/}.tar.gz
+
+      - name: Update PKGBUILD
+        run: |
+          sed -i "s/tag=/tag=${GITHUB_REF##*/}/g" build/PKGBUILD
+          sed -i "s/sha256sums=/sha256sums=('$(sha256sum ${GITHUB_REF##*/}.tar.gz | awk '{print $1;}')')/g" build/PKGBUILD
```
Comment (williamleven) [inline .github/workflows/publish-aur.yml:21]: Would be nice if this could be split up or commented some way

File: build/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+source=($url/archive/$_tag.tar.gz)
+sha256sums=
+# The name of the root directory of the tar.gz that Github
+# automatically creates and which therefore will be extracted
+_src="BooGroCha-${pkgver:1}"
+
+build() {
+    cd $_src
+    # Build as per Go package guidelines
+    go build \
```
Comment (williamleven) [inline build/PKGBUILD:24]: what does all this stuff do?

File: build/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#Maintainer: Karl Wikström <boogrocha@sidus.io>
+#Maintainer: William Leven <boogrocha@sidus.io>
+
```
Comment (williamleven) [inline build/PKGBUILD:3]: maybe add a comment about tag and checksums being populated by pipeline?

**Review by williamleven — APPROVED** [review]
Well done! Have placed a few comments, but nothing that stops you from merging. But please squash when you merge!

File: build/PKGBUILD (hunk 2 above)
Comment (HerrNaN) [reply build/PKGBUILD:24]: https://wiki.archlinux.org/index.php/Go_package_guidelines#Flags_and_build_options

**Pull request merged**